src/Stratis.Bitcoin/Controllers/RestApiClientBase.cs

```diff
@@ -106,6 +106,8 @@ public RestApiClientBase(ILoggerFactory loggerFactory, IHttpClientFactory httpCl
         public async Task<Response> SendGetRequestAsync<Response>(string apiMethodName, string arguments = null,
             CancellationToken cancellation = default(CancellationToken)) where Response : class
         {
+            this.logger.LogTrace("{0}:{1}, {2}:{3}", nameof(apiMethodName), apiMethodName, nameof(arguments), arguments);
```

Comment:
we already have this with fody enabled. No need to add it manually.

Reply:
OK yeah will remove!